extensions/2.0/Khronos/KHR_xmp/README.md

```diff
@@ -0,0 +1,322 @@
+XMP metadatas are defined within a dictionary property in the glTF scene description file, by adding an extensions property to the top-level glTF 2.0 object and defining a KHR_xmp property with a `packets` array inside it.
+The following example defines a glTF scene with a sample XMP metadata.
+
+```
+"extensions": {
+    "KHR_xmp" : {
+        "packets" : [
+            {
+                "dc:contributor" : [ "Creator1Name", "Creator2Email@email.com", "Creator3Name<Email@email.com>"],
+                "dc:coverage" : "Bay Area, California, United States",
+                "dc:creator" : [ "CreatorName", "CreatorEmail@email.com"],
+                "dc:date" : ["1997-07-16T19:20:30+01:00"],
+                "dc:description" : { "en-us": "text"},
+                "dc:format" : "model/gltf-binary",
+                "dc:identifier" : "urn:stock-id:292930",
+                "dc:language" : ["en"],
+                "dc:publisher" : ["Company"],
+                "dc:relation" : ["https://www.khronos.org/"],
+                "dc:rights" : {"en-us": "BSD"},
+                "dc:source" : "http://related_resource.org/derived_from_this.gltf",
+                "dc:subject" : ["architecture"],
+                "dc:title" : { "en-us" : "MyModel", "it-IT" : "Mio Modello"},
+                "dc:type" : ["Physical Object"],
+                "gltf:spdx" : "BSD-Protection",
+            }
+        ]
+    }
+}
+```
+
+
+## Instantiating XMP metadata
+Metadata can be instantiated in a node by defining the `extensions.KHR_xmp` property and, within that, an index into the `packets` array using the `packet` property.
+The following example shows a glTF Mesh instantiating the XMP metadata at index `0`.
+
+```
+{
+    "meshes": [
+      {
+        "extensions" : {
+            "KHR_xmp": {
+                "packet" : 0,
+            }
+        },
+        "primitives": [
+          {
+            "attributes": {
+                "NORMAL": 23,
+                "POSITION": 22,
+                "TANGENT": 24,
+                "TEXCOORD_0": 25
+            },
+            "indices": 21,
+            "material": 3,
+            "mode": 4
+          }
+        ]
+      }
+    ]
+}
+```
+
+### Notes:
+
+#### Overriding Rule
+A glTF scene might reference resources already containing XMP metadata.
+A relevant example would be an image node referencing a PNG file with embedded XMP metadata.
+When this happens the XMP metadata specified in the glTF manifest takes precedence over the metadata embedded in the referenced resource.
+
+
+#### Transferring and merging metadata
+When a glTF object (for example a Mesh) is copied from a glTF file to another, the associated KHR_xmp metadata should be copied as well.
+Copying metadata requires adding the source object metadata packet to the `KHR_xmp.packets` extension of the copied glTF and reference it from the copied object `KHR_xmp.packet` extension.
```

Comment:
The phrasing of this second line is a little confusing. I suggest changing it to this:
"Copying metadata requires adding the source object's metadata packet to the destination glTF's `KHR_xmp.packets` array."